test/lint/pass_lint_fix_json.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+clean() { rm -rf "$TMP"; }
+trap clean EXIT
+
+cat << 'EOF' > "$TMP/schema.json"
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "string",
+  "enum": [ "foo" ]
+}
+EOF
+
+"$1" lint "$TMP/schema.json" --fix --json >"$TMP/output.json" 2>&1 && CODE="$?" || CODE="$?"
+test "$CODE" = "0" || exit 1
+
+cat << EOF > "$TMP/expected.json"
+{
+  "valid": true,
+  "errors": [],
+  "fixApplied": true
```

Comment:
Let's get rid of this flag. We shouldn't report it. Usually what linters do with `--fix` is that they will still report ALL the errors that were fixed (as if you didn't pass `--fix`) but output code 0. And if you run it again, you get no errors reported at all